06_gpu/tensorflow_tutorial.py

```diff
@@ -25,7 +25,9 @@
 stub = modal.Stub(
     "example-tensorflow-tutorial",
     image=(
-        modal.Image.conda().conda_install(["cudatoolkit=11.2", "cudnn=8.1.0"]).pip_install(["tensorflow", "pathlib"])
+        modal.Image.conda()
+        .conda_install(["cudatoolkit=11.2", "cudnn=8.1.0", "cuda-nvcc"], channels=["conda-forge", "nvidia"])
+        .pip_install(["tensorflow~=2.9.1"])
```

Comment:
Later versions of Tensorflow (2.10.x, 2.11.x), become more difficult to install with Conda: https://github.com/tensorflow/tensorflow/issues/56927.

Reply:
maybe we shouldn't use conda for it?

Reply:
Maybe, but it's the only `conda` example we have I think, and it's helpful to catch problems.